File: backend/src/main/java/um/edu/ar/backend/infrastructure/web/dto/VentaDetalleResponse.java
package um.edu.ar.backend.infrastructure.web.dto;

import java.time.Instant;
import java.util.List;

public record VentaDetalleResponse(
        Long eventoId,
        Long ventaId,
        Instant fechaVenta,
        List<AsientoVentaResponse> asientos,
        boolean resultado,
        String descripcion,
        double precioVenta
) {}

File: cliente_movil/composeApp/src/commonMain/kotlin/um/edu/ar/model/VentaDetalleResponse.kt
package um.edu.ar.model

import kotlinx.serialization.Serializable

@Serializable
data class VentaDetalleResponse(
    val eventoId: Long,
    val ventaId: Long? = null,
    val fechaVenta: String,
    val asientos: List<AsientoVentaResponse> = emptyList(),
    val resultado: Boolean,
    val descripcion: String,
    val precioVenta: Double
) {
    @Serializable
    data class AsientoVentaResponse(
        val fila: Int,
        val columna: Int,
        val persona: String,
        val estado: String
    )
}
